Error compiling Project Generator on Linux (GCC 15.2.0)

When trying to compile Project Generator on Debian Linux (Sid), GCC 15.2.0, I get the following error:

```
[...]

src/utils/LibraryBinary.cpp: In function ‘std::string findStringsInPath(const std::vector<std::__cxx11::basic_string<char> >&, const std::filesystem::__cxx11::path&)’:
src/utils/LibraryBinary.cpp:16:22: error: no matching function for call to ‘find(std::vector<std::__cxx11::basic_string<char> >::const_iterator, std::vector<std::__cxx11::basic_string<char> >::const_iterator, const std::vector<std::__cxx11::basic_string<char> >&)’
   16 |         if (std::find(strings.begin(), strings.end(), vp) != strings.end()) {
      |             ~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

[...]

```


Adding 
```
#include <algorithm>
```
 to src/utils/LibraryBinary.h fixes the error.

--- a/commandLine/src/utils/LibraryBinary.h
+++ b/commandLine/src/utils/LibraryBinary.h
@@ -1,8 +1,8 @@
 #pragma once
 
+#include <algorithm>
 #include <string>
 #include <vector>
-
 
 #include "ofConstants.h"
 #include "defines.h"

